themes/bootstrap3/templates/Recommend/SideFacets.phtml

```diff
@@ -7,238 +7,91 @@
 <? if ($results->getResultTotal() > 0): ?>
   <h4><?=$this->transEsc(isset($this->overrideSideFacetCaption) ? $this->overrideSideFacetCaption : 'Narrow Search')?></h4>
 <? endif; ?>
-<? $checkboxFilters = $results->getParams()->getCheckboxFacets(); if (count($checkboxFilters) > 0): ?>
-<?
-  $html = '';
-  $shown = 0;
-  foreach ($checkboxFilters as $current) {
-    $html .= '<label class="checkbox';
-    if($results->getResultTotal() < 1 && !$current['selected'] && !$current['alwaysVisible']) {
-      $html .= ' hidden';
-    } else {
-      $shown ++;
+<? $checkboxFilters = $results->getParams()->getCheckboxFacets(); ?>
+<? if (count($checkboxFilters) > 0):
+    $checkboxesShown = false;
```

Comment:
You should initialize $checkboxesShown before the if statement to ensure that it is defined below.